docs/pages/enroll-resources/mcp-access/mcp-access.mdx

```diff
@@ -1,30 +1,74 @@
 ---
 title: MCP Servers
-description: Guides to protecting MCP servers with Teleport
+description: Protect MCP servers with Teleport's access controls, identity governance, and auditing capabilities 
+template: doc-page
 tags:
  - mcp
  - zero-trust
 ---
 
-Teleport can provide secure connections to your MCP (Model Context Protocol)
-servers while improving both access control and visibility.
+import DocHero from "@site/src/components/Pages/Landing/DocHero";
+import UseCasesList from "@site/src/components/Pages/Landing/UseCasesList";
 
-![Architecture diagram for enrolling MCP servers with Teleport](../../../img/mcp-access/architecture.svg)
+import userCheckSvg from "@site/src/components/Icon/svg/user-check.svg";
+import questionSvg from "@site/src/components/Icon/svg/question2.svg";
 
+<DocHero
+  title="Enroll your first MCP server"
+  youtubeVideoId="JdiNX4pIUaU"
+  ctaLinks={[
+    {
+      title: "Get started with MCP servers",
+      href: "./getting-started/",
+      variant: "secondary",
+      arrow: true,
+    },
+  ]}
+  links={[
+    {
+      title: "MCP Access Controls",
+      description: "Use Teleport's role-based access control (RBAC) system to set up granular permissions for authenticating to MCP servers connected to Teleport.",
+      href: "./rbac/",
+      icon: userCheckSvg,
+    },
+    {
+      title: "Troubleshooting",
+      description: "Understand common issues that you might encounter in managing access to MCP servers with Teleport and how to work around or resolve them.",
+      href: "./troubleshooting/",
+      icon: questionSvg,
+    },
+  ]}
+>
+{/* vale messaging.protocol-products = NO */}
+Connect an MCP server with the Teleport Application Service, configure and assign a role granting MCP permissions, and connect and query the server with the Teleport CLI or your AI platform of choice.
+</DocHero>
 
-## Getting started
-- [Quick-start with Demo MCP Server](./getting-started.mdx): Provides instructions to set up a Teleport Service and enable secure access to a demo MCP server.
-
-## Guides
-- [MCP Access with Streamable-HTTP MCP Server](./streamable-http.mdx): Set up a Teleport Service and enable secure access to a MCP server with streamable-HTTP transport.
```

Comment:
@rodgtr1 as shown here, a few new pages have been added recently:
- MCP Access with Streamable-HTTP MCP Server
- MCP Access with SSE MCP Server
-JWT Authentication to MCP server

more details: see https://github.com/gravitational/teleport/pull/60410

Reply:
Thanks for alerting us to this!